src/utils/responsiveness.cljs

```diff
@@ -0,0 +1,10 @@
+(ns utils.responsiveness)
+
+(def ^:const IPHONE_11_PRO_VIEWPORT_WIDTH 375)
+
+(defn iphone-11-Pro-20-pixel-from-width
```

Comment:
👍

Reply:
Personally I would put the comments on top of the function to explain it's use and not inside. like a docstring type comment